src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedEditorFormPageObject.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.imageupload.UploadImageModalComponentObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class CuratedEditorFormPageObject extends BasePageObject {
+
+  @FindBy(css = "input#label")
+  private WebElement displayNameField;
+  @FindBy(css = ".curated-content-editor-photo")
+  private WebElement imageField;
+
+  @FindBy(css = ".sub-head--cancel")
+  private WebElement backButton;
+  @FindBy(css = ".sub-head--done")
```

Comment:
this selector is the same almost all editor page objects? same goes for method which clicks it